solidity/package.json

```
@@ -21,8 +21,8 @@
     "lint:fix": "npm run lint:fix:js && npm run lint:fix:sol",
     "lint:js": "eslint ${npm_package_config_eslintPaths}",
     "lint:fix:js": "eslint --fix ${npm_package_config_eslintPaths}",
-    "lint:sol": "solium -d contracts/ && prettier --list-different **/*.sol",
-    "lint:fix:sol": "solium -d contracts/ --fix && prettier --write **/*.sol"
+    "lint:sol": "solium -d contracts/ && prettier --list-different contracts/ **/*.sol",
+    "lint:fix:sol": "solium -d contracts/ --fix && prettier --write contracts/ **/*.sol"
```

```suggestion
    "lint:sol": "solium -d contracts/ && prettier --check '**/*.sol'",
    "lint:fix:sol": "solium -d contracts/ --fix && prettier --write '**/*.sol'"
```
See:
https://github.com/keep-network/keep-core/pull/2244#discussion_r556629670
https://github.com/keep-network/keep-core/pull/2244#discussion_r556625475

---

Done a13b562